tests/python/dnnl/subgraphs/test_conv_subgraph.py

```diff
@@ -115,6 +115,45 @@ def forward(self, x):
   check_fusion(net, data_shape, attr)
 
 
+@mx.util.use_np
+@pytest.mark.parametrize('data_shape', DATA_SHAPE)
+@pytest.mark.parametrize('no_bias', [True, False])
+@pytest.mark.parametrize('out_type', ['int8', 'auto'])
```

Comment:
Why not uint8?

Reply:
I checked other tests from this file and they only test int8 and auto types, @anko-intel should I add uint8 as well?

Reply:
Following settings for other tests in this file, I do not test it as it is scenario which is not used.